src/ploigos_step_runner/step_implementers/shared/openscap_generic.py

```diff
@@ -28,6 +28,9 @@
                                                        remote resources and this is not True. \
                                                        For disconnected environments the remote \
                                                        internal mirror.
+'oscap-severity'               | No        | ''      | Severity threshold for failing a step. \
```

Comment:
can you add the expected/allowed values here

Reply:
I'll get that in tomorrow.